### PR TITLE
Validation start screen (EXPOSUREAPP-8030)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
Validation start screen + multiple urls in one text extension (big cheers to @d4rken 🍺 )

Screenshot 2021-07-02 at 15 54 40

How to test:
1) Scan test certificate
2) Go to certificate details
3) Press "Gültigkeit prüfen" at the bottom of the screen (would be replaced with correct button in upcoming PR)
4) Check new screen

TODO: Wire things up with some server check magic

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationstart/ValidationStartViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    data class UIState(
+        val dccCountry: DccCountry? = null,
+        val date: DateTime = DateTime.now(),
+    ) {
+        fun getDate(locale: Locale) = getFormattedTime(date, locale)
+
+        private fun getFormattedTime(value: DateTime?, locale: Locale) =
+            value?.toString("${locale.shortDatePattern()} HH:mm", locale)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationstart/ValidationStartViewModel.kt:64]: the date format is different in Figma

File: Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="6dp"
+                android:layout_marginEnd="@dimen/spacing_normal"
+                android:text="@string/validation_start_date_info"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toStartOf="@id/body"
+                app:layout_constraintTop_toBottomOf="@id/date_layout" />
+
+            <ImageView
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml:124]: Please check the spacing between the icon and text ? in Figma looks a little bit bigger than here

File: Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="22dp"
+                android:layout_marginBottom="@dimen/spacing_tiny"
+                android:hint="@string/validation_start_land_hint"
+                android:textColorHint="@color/colorTextPrimary3"
+                app:boxBackgroundColor="@color/colorContactDiaryListItem"
+                app:layout_constraintStart_toStartOf="@+id/body"
+                app:layout_constraintTop_toBottomOf="@+id/subtitle">
+
+                <AutoCompleteTextView
```
[the remark this answers is not in the document]
Comment (mtwalli) [reply Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml:76]: Highlighted colour is grey , in Figma is blue

File: Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+                android:layout_height="wrap_content"
+                android:layout_marginTop="26dp"
+                android:text="@string/validation_start_certificate_proof"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toStartOf="@id/body"
+                app:layout_constraintTop_toBottomOf="@id/date_info" />
+
+            <TextView
+                android:id="@+id/note_subtitle"
+                style="@style/TextAppearance.MaterialComponents.Subtitle2"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml:150]: Text size or Text style is somehow bigger/different in Figma

File: Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="24dp"
+                android:text="@string/validation_start_note_subtitle"
+                app:layout_constraintEnd_toEndOf="@id/body"
+                app:layout_constraintStart_toStartOf="@id/body"
+                app:layout_constraintTop_toBottomOf="@id/certificate_proof" />
+
+            <include
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/validation_start_fragment.xml:159]: bullets alignment should be shifted a little bit , currently it is aligned to the title

**Review by mtwalli — COMMENTED** [review]
Looks good 👍

**Review by BMItr — APPROVED** [review]
👌

**Review by mtwalli — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**